core/arch/arm/kernel/entry_a64.S

```diff
@@ -479,7 +479,7 @@ END_FUNC unhandled_cpu
 
 	.section .identity_map, "ax", %progbits
 	.align	11
-LOCAL_FUNC reset_vect_table , :, .identity_map
+LOCAL_FUNC reset_vect_table , :, .identity_map, , naked
```

Comment:
Minor: I suggest using BTI specific terms. Trying to generalize too soon makes the code harder to read in the simple case where only one scenario is supported. We can always extend/rework later. Hence, I suggest `s/naked/nobti/`.